src/medVtkInria/vtkDataManagement/vtkMetaDataSet.h

```diff
@@ -365,6 +367,7 @@ class MEDVTKINRIA_EXPORT vtkMetaDataSet: public vtkDataObject
   virtual void SetScalarNullValue(const char * arrayName, double nullValue);
 
   virtual double* GetCurrentScalarRange();
+  virtual double* GetCurrentScalarRange(const QString & attributeName);
```

Comment:
This doesn't get the **current** scalar array but the one named ```attributeName``` so the method name is misleading.